src/shipit_code_coverage_backend/requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+-r ./../../lib/backend_common/requirements-dev.txt
+
+aresponses
+responses
```

Comment:
IIRC we're not using `responses` anymore, can you check in the tests of shipit_uplift?